src/components/Proposal/SentimentSurvey/SentimentSurvey.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Loader } from 'decentraland-ui/dist/components/Loader/Loader'
+
+import { ReactionType, Survey, SurveyTopicAttributes, Topic } from '../../../entities/SurveyTopic/types'
+
+import './SentimentSurvey.css'
+import SentimentSurveyRow from './SentimentSurveyRow'
+
+interface Props {
+  survey: Survey
+  surveyTopics: Pick<SurveyTopicAttributes, 'label' | 'topic_id'>[] | null
+  isLoadingSurveyTopics: boolean
+  setSurvey: React.Dispatch<React.SetStateAction<Survey>>
+}
+
+const SentimentSurvey = ({ survey, surveyTopics, isLoadingSurveyTopics, setSurvey }: Props) => {
+  const t = useFormatMessage()
+
+  useEffect(() => {
+    if (!isLoadingSurveyTopics) {
```

Comment:
I think you can group these two conditions
`if (!isLoadingSurveyTopics && survey.length === 0) {`